include/rocksdb/stats_history.h

```diff
@@ -11,19 +11,25 @@
 #include <map>
 #include <string>
 
-// #include "db/db_impl.h"
 #include "rocksdb/statistics.h"
 #include "rocksdb/status.h"
 
 namespace rocksdb {
 
 class DBImpl;
 
+// StatsHistoryIterator is the main interface for users to programmatically
+// access statistics snapshots that was automatically stored by RocksDB.
+// Depending on options, the stats can be in memory or on disk.
+// The stats snapshots are indexed by time that they were recorded, and each
+// stats snapshot is an std::map from individual stat name to an integer value
+// of the stat at the time of recording.
```

Comment:
Can you provide example code how to use it here, as suggested in https://google.github.io/styleguide/cppguide.html#Class_Comments?